test/test_gipc.py

```diff
@@ -18,19 +18,18 @@
 import logging
 import multiprocessing
 
-
 import gevent
 import gevent.queue
+import gevent.signal
```

Comment:
Remove this import, and then I think we're good.

The meaning of `gevent.signal` _changes_ after `import gevent.signal`, I think.

Reply:
yeah it does change :/

Reply:
Looks like they all pass now, I'll re-squash to remove the cruft.